.chloggen/add-support-cloud-armor-logs.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. receiver/filelog)
+component: extension/encoding
```

```suggestion
component: extension/googlecloudlogentry_encoding
```

See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/604630c20cf84d05771b215772f016a9c2ab9abf/.chloggen/config.yaml#L97C7-L97C45